memberlist.go

```diff
@@ -776,3 +777,17 @@ func (m *Memberlist) changeNode(addr string, f func(*nodeState)) {
 	n := m.nodeMap[addr]
 	f(n)
 }
+
+// checkQueueDepth periodically checks the size of a queue to see if
+// it is too large
+func (m *Memberlist) checkQueueDepth(name string, queue *TransmitLimitedQueue) {
```

Comment:
I'm not sure if we need a name and queue as parameters in the method. Since memberlist struct only has one queue.

Reply:
See above comment this has been resolved.  Thank you for calling this out.

Reply:
Looks great. Thanks for the update.